tests/data_context/test_data_context_test_yaml_config.py

```diff
@@ -1092,3 +1092,181 @@ def test_golden_path_configured_asset_pandas_datasource_configuration(
 
     # No other usage stats calls detected
     assert mock_emit.call_count == 1
+
+
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+def test_golden_path_runtime_data_connector_pandas_datasource_configuration(
+    mock_emit, empty_data_context_stats_enabled, test_df, tmp_path_factory
+):
+    """
+    Tests output of RuntimeDataConnector
+    """
```

Comment:
Could we add a bit more detail to this test description? Maybe note that we are testing the output of `test_yaml_config` to ensure that it yields the appropriate `report_object` with an additional note specific to `RuntimeDataConnectors`?